common/src/allocations/monitor.rs

```diff
@@ -111,12 +81,13 @@ pub fn indexer_allocations(
     // Refresh indexer allocations every now and then
     timer(interval).map_with_retry(
         move |_| async move {
-            let current_epoch = current_epoch(network_subgraph, graph_network_id)
-                .await
-                .map_err(|e| format!("Failed to fetch current epoch: {}", e))?;
-
-            // Allocations can be closed one epoch into the past
-            let closed_at_epoch_threshold = current_epoch - 1;
+            // Allocations can be closed 5 mins into the past
+            let start = SystemTime::now();
+            let since_the_epoch = start
+                .duration_since(UNIX_EPOCH)
+                .expect("Time went backwards");
+            const MINUTES_THRESHOLD: u64 = 5;
```

Comment:
Perhaps are safer default could be 1 hour? (yes sorry I know I gave the example of 5min earlier today).
Could it make sense to have this a program parameter?

Reply:
Well, in that case we'd need to have a parameter for both indexer-service and tap-agent.

Reply:
Just added a config for both indexer-service and tap-agent. The default value is 1 hour.